### PR TITLE
Perform s3cmd get for some large and small objects resides in bucket

### PR DESCRIPTION
**Pull request by anrao19**
https://issues.redhat.com/browse/RHCEPHQE-9484

LOG: http://magna002.ceph.redhat.com/ceph-qe-logs/Anuchaithra/test_get_s3cmd.console.log

Step | Expected Result
create a cluster --> should succeed
upload objects to the bucket, few objects should be of size > 19M -->should succeed
Perform s3cmd get on the objects > 19M --> should succeed ,should not fail with EOF error

### CODE REVIEW

File: rgw/v2/tests/s3cmd/test_s3cmd.py [hunk 1]
```diff
@@ -61,6 +61,56 @@ def test_exec(config, ssh_con):
...
+            bucket_name = utils.gen_bucket_name_from_userid(
+                user_info[0]["user_id"], rand_no=bc
+            )
+            s3cmd_reusable.create_bucket(bucket_name)
+            log.info(f"Bucket {bucket_name} created")
+            s3cmd_path = "/home/cephuser/venv/bin/s3cmd"
+            object_count = config.objects_count // 2
+
+            log.info(f"uploading some large objects to bucket {bucket_name}")
+            utils.exec_shell_cmd(f"fallocate -l 20m obj20m")
```
Comment (hmaheswa) [inline rgw/v2/tests/s3cmd/test_s3cmd.py:82]: @anrao19 , can we get these object sizes from the config?

Comment (anrao19) [reply rgw/v2/tests/s3cmd/test_s3cmd.py:82]: @hmaheswa , we do not have provision to provide size from config for s3md, we can go a head with this method for now.
we can take this as enhancement

**Review by hmaheswa — APPROVED** [review]
LGTM

File: rgw/v2/tests/s3cmd/test_s3cmd.py [hunk 2]
```diff
@@ -61,6 +61,56 @@ def test_exec(config, ssh_con):
 
     ip_and_port = s3cmd_reusable.get_rgw_ip_and_port(ssh_con)
 
+    if config.test_ops.get("s3cmd_get_objects", False):
+        log.info(
+            f"CEPH-83575477 - Verify s3cmd get: Bug 2174863 - [cee/sd][RGW] 's3cmd get' fails with EOF error for few objects"
+        )
```
Comment (mkasturi18) [inline rgw/v2/tests/s3cmd/test_s3cmd.py:67]: @anrao19 , please modify the log.info to reflect object get, we can add testcase details as a comment.

File: rgw/v2/tests/s3cmd/test_s3cmd.py [hunk 3]
```diff
@@ -61,6 +61,56 @@ def test_exec(config, ssh_con):
...
+        user_info = resource_op.create_users(no_of_users_to_create=1)
```
Comment (mkasturi18) [inline rgw/v2/tests/s3cmd/test_s3cmd.py:68]: @anrao19 , could you fetch the user count  details from conf instead default value

Comment (anrao19) [reply rgw/v2/tests/s3cmd/test_s3cmd.py]: made changes @mkasturi18

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**